packages/rum/src/types.ts

```diff
@@ -12,6 +12,7 @@ export interface SegmentMeta extends SegmentContext {
   has_full_snapshot: boolean
   records_count: number
   creation_reason: CreationReason
+  [key: string]: any
```

Comment:
I'd prefer a `as Context` cast when using it in `addMonitoringMessage`, so we keep a clean type for the rest of the application